docs/docs/cloud/how-tos/human_in_the_loop_edit_state.md

```diff
@@ -129,10 +176,22 @@ Now, let's assume we actually meant to search for the weather in Sidi Frej (anot
     await client.threads.updateState(thread['thread_id'], {values:{"messages": lastMessage}});
     ```
 
+=== "CURL"
+
+    ```bash
+    curl --request GET --url whatever-your-deployment-url-is/threads/_YOUR_THREAD_ID_/state | \                                                                                      
```

Comment:
(nit) i wonder if we can use a consistent format for this everywhere, something like <DEPLOYMENT_URL> and <THREAD_ID> -- i think that's common for placeholders